src/main/java/com/dumptruckman/bukkit/configuration/json/JsonConfiguration.java

```diff
@@ -110,10 +105,8 @@ private static JsonConfiguration loadConfiguration(@NotNull final JsonConfigurat
             config.load(file);
         } catch (FileNotFoundException ex) {
             LOG.log(Level.SEVERE, "Cannot find file " + file, ex);
-        } catch (IOException ex) {
+        } catch (IOException | InvalidConfigurationException ex) {
```

Comment:
Hmm... This works cause you're building with something later than Java 8 I imagine. But is it actually compatible with Java 8 like in the build.gradle?

Reply:
![image](https://github.com/user-attachments/assets/36502c2a-f592-4160-92de-180ba27012c4)
My current build config is using 1.8 and it can compile.
I googled, its a java 7 feature, so we are safe.